vote_text_drafts/rpip-28.md

```diff
@@ -0,0 +1,29 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# Deposits Under the Minimum (RPIP-28)
```

Comment:
Note, check my terminology around minipools and nodes, I'm unfamiliar with the common terms.